MantidQt/CustomInterfaces/test/ReflSaveTabPresenterTest.h

```diff
@@ -33,7 +34,7 @@ class ReflSaveTabPresenterTest : public CxxTest::TestSuite {
   }
   static void destroySuite(ReflSaveTabPresenterTest *suite) { delete suite; }
 
-  ReflSaveTabPresenterTest() {}
+  ReflSaveTabPresenterTest() { FrameworkManager::Instance(); }
```

Comment:
It does seem quite necessary for creating and running algorithms used in the tests. It's staying.